builder/vsphere/clone/step_customize.go

```diff
@@ -202,24 +202,24 @@ func (c *CustomizeConfig) Prepare() ([]string, []error) {
 		errs = append(errs, fmt.Errorf("one or more `network_interface` must be provided"))
 	}
 
-	options_number := 0
+	optionsNumber := 0
 	if c.LinuxOptions != nil {
-		options_number = options_number + 1
+		optionsNumber = optionsNumber + 1
 	}
 	if c.WindowsOptions != nil {
-		options_number = options_number + 1
+		optionsNumber = optionsNumber + 1
 	}
 	if c.WindowsSysPrepFile != "" {
 		warnings = append(warnings, windowsSysprepFileDeprecatedMessage)
-		options_number = options_number + 1
+		optionsNumber = optionsNumber + 1
 	}
 	if c.WindowsSysPrepText != "" {
-		options_number = options_number + 1
+		optionsNumber = optionsNumber + 1
 	}
 
-	if options_number > 1 {
+	if optionsNumber > 1 {
 		errs = append(errs, errCustomizeOptionMutualExclusive)
-	} else if options_number == 0 {
+	} else if optionsNumber == 0 {
 		errs = append(errs, fmt.Errorf("one of `linux_options`, `windows_options`, `windows_sysprep_file` must be set"))
```

Comment:
Looks like the error message doesn't mention the `windows_sysprep_text` option here?

Reply:
Good catch! Resolved in c5db4fe.